### ISSUE TITLE
Any plans to make new release?

### ISSUE THREAD

**Issue by kloczek** [NLnetLabs/ldns#120]
I think that it would be good to flush currently committed changes and make new release :)

**Comment by wtoorop** [comment]
Resolved in ldns-1.8.0

**Comment by kloczek** [comment]
Just tested 1.9.0 and found small issue in install target.
```console
+ cd ldns-1.8.0
+ /usr/bin/make install DESTDIR=/home/tkloczko/rpmbuild/BUILDROOT/ldns-1.8.0-2.fc35.x86_64 'INSTALL=/usr/bin/install -p' install-doc install-pc
/usr/bin/install -p -m 755 -d /home/tkloczko/rpmbuild/BUILDROOT/ldns-1.8.0-2.fc35.x86_64/usr/include/ldns
for i in buffer.h dane.h dname.h dnssec.h dnssec_sign.h dnssec_verify.h dnssec_zone.h duration.h error.h higher.h host2str.h host2wire.h keys.h ldns.h packet.h parse.h radix.h rbtree.h rdata.h resolver.h rr_functions.h rr.h sha1.h sha2.h str2host.h tsig.h update.h wire2host.h zone.h; do \
        /usr/bin/install -p -c -m 644 ./ldns/$i /home/tkloczko/rpmbuild/BUILDROOT/ldns-1.8.0-2.fc35.x86_64/usr/include/ldns/; done
for i in common.h util.h net.h; do \
        /usr/bin/install -p -c -m 644 ldns/$i /home/tkloczko/rpmbuild/BUILDROOT/ldns-1.8.0-2.fc35.x86_64/usr/include/ldns/; done
/usr/bin/install -p -m 755 -d /home/tkloczko/rpmbuild/BUILDROOT/ldns-1.8.0-2.fc35.x86_64/usr/lib64
./libtool --tag=CC --quiet --mode=install cp libldns.la /home/tkloczko/rpmbuild/BUILDROOT/ldns-1.8.0-2.fc35.x86_64/usr/lib64
libtool: warning: remember to run 'libtool --finish /usr/lib64'
./libtool --tag=CC --quiet --mode=finish /home/tkloczko/rpmbuild/BUILDROOT/ldns-1.8.0-2.fc35.x86_64/usr/lib64
/usr/bin/install -p -m 644 ./packaging/libldns.pc /home/tkloczko/rpmbuild/BUILDROOT/ldns-1.8.0-2.fc35.x86_64/usr/lib64/pkgconfig/ldns.pc
/usr/bin/install: cannot create regular file '/home/tkloczko/rpmbuild/BUILDROOT/ldns-1.8.0-2.fc35.x86_64/usr/lib64/pkgconfig/ldns.pc': No such file or directory
```
Looks like below patch should fix the issue
```patch
--- a/Makefile.in~      2021-11-26 08:24:28.000000000 +0000
+++ b/Makefile.in       2021-11-26 14:39:20.881657121 +0000
@@ -397,7 +397,7 @@
        ./config.status --file=$@

 install-pc: packaging/libldns.pc
-       $(INSTALL) -m 644 $(srcdir)/packaging/libldns.pc $(DESTDIR)$(libdir)/pkgconfig/ldns.pc
+       $(INSTALL) -Dm644 $(srcdir)/packaging/libldns.pc $(DESTDIR)$(libdir)/pkgconfig/ldns.pc

 uninstall-pc:
        $(LIBTOOL) --mode=uninstall rm -f $(DESTDIR)$(libdir)/pkgconfig/ldns.pc
```
In meantime I've switches to autoconf 2.71 and with that version autoreconf shows some warnings:
```console
+ autoreconf -fiv
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: aclocal --force
autoreconf: running: /usr/bin/autoconf --force
configure.ac:35: warning: The macro `AC_AIX' is obsolete.
configure.ac:35: You should run autoupdate.
./lib/autoconf/specific.m4:540: AC_AIX is expanded from...
configure.ac:35: the top level
configure.ac:88: warning: The macro `AC_LANG_C' is obsolete.
configure.ac:88: You should run autoupdate.
./lib/autoconf/c.m4:72: AC_LANG_C is expanded from...
configure.ac:88: the top level
configure.ac:151: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:151: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:151: the top level
configure.ac:176: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:176: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:176: the top level
configure.ac:201: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:201: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:201: the top level
configure.ac:216: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:216: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:216: the top level
configure.ac:236: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:236: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:236: the top level
configure.ac:292: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:292: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:292: the top level
configure.ac:323: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:323: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:323: the top level
configure.ac:389: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:389: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:389: the top level
configure.ac:514: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:514: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:514: the top level
configure.ac:526: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:526: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:526: the top level
configure.ac:544: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:544: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:544: the top level
configure.ac:562: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:562: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:562: the top level
configure.ac:578: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:578: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:578: the top level
configure.ac:595: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:595: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:595: the top level
configure.ac:612: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:612: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:612: the top level
configure.ac:613: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:613: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:613: the top level
configure.ac:614: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:614: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:614: the top level
configure.ac:670: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:670: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:670: the top level
configure.ac:678: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:678: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:678: the top level
configure.ac:686: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:686: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:686: the top level
configure.ac:694: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:694: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:694: the top level
configure.ac:702: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:702: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:702: the top level
configure.ac:710: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:710: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:710: the top level
configure.ac:718: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:718: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:718: the top level
configure.ac:726: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:726: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:726: the top level
configure.ac:775: warning: The macro `AC_HEADER_STDC' is obsolete.
configure.ac:775: You should run autoupdate.
./lib/autoconf/headers.m4:704: AC_HEADER_STDC is expanded from...
configure.ac:775: the top level
configure.ac:955: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:955: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:955: the top level
configure.ac:984: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:984: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:984: the top level
configure.ac:991: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:991: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:991: the top level
configure.ac:1001: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:1001: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:1001: the top level
configure.ac:1182: warning: The macro `AC_CONFIG_HEADER' is obsolete.
configure.ac:1182: You should run autoupdate.
./lib/autoconf/status.m4:719: AC_CONFIG_HEADER is expanded from...
configure.ac:1182: the top level
autoreconf: running: /usr/bin/autoheader --force
autoreconf: configure.ac: not using Automake
autoreconf: './config.sub' is created
autoreconf: './config.guess' is created
autoreconf: Leaving directory '.'
```
